### ISSUE TITLE
User page UI: show when Bluesky repo is deactivated/tombstoned

### ISSUE THREAD

**Issue by comodoropet4** [snarfed/bridgy-fed#1330]
Greetings!

I've followed the _Bridgy Fed for Bluesky_ account with my _uwu.social_ profile on _Mastodon_, and allowed it to follow me back to engage the federation process with _Bluesky_.

However, when I look up my profile over there, it doesn't exist. I search it up, no dice. I click on the link to it from the confirmation DM I got from the Bridgy Fed account, and the Bluesky app tells me the account doesn't exist.

![Screenshot_20240914_135234_Bluesky.jpg](https://github.com/user-attachments/assets/9090fe0d-f52f-4efe-8b08-f123e0cc51f5)

My user page on bridgy fed shows that the federation has been made, and it even has a log of my public activity from Mastodon that should be federated on Bluesky.

![Screenshot_20240914_134731_Edge.jpg](https://github.com/user-attachments/assets/d6e8130c-6c1b-42ee-b02e-24611ab68531)

I hope it's not a big ask to request help on getting this issue solved.

Thanks!

**Comment by snarfed** [comment]
Hey, sorry for the trouble! Have you looked at https://fed.brid.gy/docs#troubleshooting ?

**Comment by comodoropet4** [comment]
> Hey, sorry for the trouble! Have you looked at https://fed.brid.gy/docs#troubleshooting ?

It's alright! I know it's a one-man effort, per the same information at the bottom of the docs, so I hope it's all good 😊

I've indeed gone through all the steps and nothing has helped me.

- My Mastodon account is more than two weeks old and has a profile picture.
- I contacted the uwu.social admin and he told me that neither bsky.brid.gy nor brid.gy were being moderated at all. I checked the moderated servers section myself and indeed they don't appear on the Moderated Servers section.

Other than that, and the obvious considerations of not blocking bridgy fed and stuff, I have nothing else I can try from my end.

**Comment by qazmlp** [comment]
Interestingly, trying to view the account in ATProto Browser results in an internal server error: https://atproto-browser.vercel.app/at/katakislives.uwu.social.ap.brid.gy
(as opposed to not finding the account at all).

The lookup for my own bridged account works fine (but notably shows an empty TODO collection).

**Comment by snarfed** [comment]
Aha, @comodoropet4 looks like you blocked @bsky.brid.gy@bsky.brid.gy at 2024-09-07 23:14:13 UTC, activity id https://uwu.social/112a55b8-b158-4680-826a-3c2cd1c50bc9 . That disables the bridge for your account, and sadly we can't yet undo that for fediverse => Bluesky, even if you unblock and refollow. https://fed.brid.gy/docs#opt-out

Sorry for the bad news. Hopefully this will be undoable eventually, feel free to follow https://github.com/snarfed/bridgy-fed/issues/1130 .

Ideally we'd show this on your Bridgy Fed user page! I'll track that UI update in this issue.

**Comment by comodoropet4** [comment]
> Aha, @comodoropet4 looks like you blocked @bsky.brid.gy@bsky.brid.gy at 2024-09-07 23:14:13 UTC, activity id https://uwu.social/112a55b8-b158-4680-826a-3c2cd1c50bc9 . That disables the bridge for your account, and sadly we can't yet undo that for fediverse => Bluesky, even if you unblock and refollow. https://fed.brid.gy/docs#opt-out
> 
> Sorry for the bad news. Hopefully this will be undoable eventually, feel free to follow https://github.com/snarfed/bridgy-fed/issues/1130 .
> 
> Ideally we'd show this on your Bridgy Fed user page! I'll track that UI update in this issue.

Well, that's unfortunate 😢

Indeed I blocked the Bridgy Fed account once in an attempt to make it work, since I tried to engage the federation process back when my account was less than two weeks old.

Alas, I'll follow the suggested issue to stay up on whether it'll be fixed.

Thanks for letting me know! 😊

**Comment by Tamschi** [comment]
The original cause of this problem (tombstoning the DID) is now fixed, but stuck accounts aren't generally reactivated yet.
There's been some progress on that starting at https://github.com/snarfed/bridgy-fed/issues/1130#issuecomment-2430525165, but it's likely this requires fixing https://github.com/snarfed/arroba/issues/34 first.

Solving that in general may obsolete this issue.

**Comment by snarfed** [comment]
Yes! Right now, if you don't have a specific protocol like Bluesky enabled, we don't show the visible link to it on your user page. That's probably enough for this.

I'm going to go ahead and close this and file a new issue for fixing tombstoned accounts that tried (unsuccessfully) to re-enable themselves.
